src/de/gurkenlabs/litiengine/gui/screens/ScreenManager.java

```diff
@@ -191,35 +338,51 @@ public Screen get(String screenName) {
 
   /**
    * Gets the currently active screen that is being rendered by the <code>RenderComponent</code>.
-   * 
+   *
+   * @deprecated   There may now be more than one 'current' screen.
+   *              currentScreen has been replaced with {@link #activeScreens}.
+   *              Please use {@link #getActiveScreens()} instead.
+   *              Currently redirects to {@link #getActiveScreens()}[0]
+   *
    * @return The currently active screen.
    * 
    * @see GameWindow#getRenderComponent()
    * @see RenderComponent#render()
    */
+  @Deprecated
   public Screen current() {
-    return this.currentScreen;
+    return this.getActiveScreens()[0];

```

Comment:
This will throw an an ArrayIndexOutOfBoundsException if there are no active screens. Code built in previous versions will be expecting `null` instead of ArrayIndexOutOfBoundsException.